bot.js

```diff
@@ -142,6 +146,7 @@ if (fs.existsSync('polldata.json')) {
 client.on('loggedOn', function(details) {
 	console.log(`Logged into Steam as ${client.steamID.getSteam3RenderedID()}`);
  	client.setPersona(SteamUser.Steam.EPersonaState.Online,config.botname);
+	client.gamesPlayed([730])
```

Comment:
@Arze1 This one could've been made better and integrated better with config file